gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -29,20 +29,20 @@ use futures_util::stream::StreamExt;
 use hex::ToHex;
 use lightning_invoice::{Currency, InvoiceBuilder, PaymentSecret};
 use ln_gateway::envs::FM_CLN_EXTENSION_LISTEN_ADDRESS_ENV;
-use ln_gateway::lightning::extension::{
```

Comment:
Why the rename? I don't think these should be in `ln_gateway::rpc`, since this is a Lightning specific thing, not an "rpc" concept (which up until now we have used only for the HTTP webserver)